Sources/UIView.swift

```diff
@@ -237,6 +237,12 @@ open class UIView: UIResponder, CALayerDelegate, UIAccessibilityIdentification {
 
     open func layoutSubviews() {
         needsLayout = false
+        
+        if layer.previousBounds != nil && layer.previousBounds != layer.bounds {
```

Comment:
Is there no "layouting step" in our render code?
Does layout occur entirely in "user space", meaning it's possible for users to prevent layout on entire subtrees of views? If so, that also doesn't seem quite right, but maybe it's the case in iOS too.

Reply:
I had a look and we do have a "non-userspace" layouting function, which would be a more appropriate place for this, if we went with this option: `Sources/UIView+SDL.swift`

Reply:
But as discussed on Slack, I think a Set containing the updated layers that gets checked and cleared before this layouting step even begins would be the correct place for this